avm/res/container-service/managed-cluster/main.bicep

```diff
@@ -64,6 +64,8 @@ param managedOutboundIPCount int = 0
 @allowed([
   'loadBalancer'
   'userDefinedRouting'
+  'managedNATGateway'
```

Comment:
The description for this property no longer reflects the new egress methods

Reply:
Good catch, fixing now.

Reply:
Fixed: https://github.com/Azure/bicep-registry-modules/pull/992/commits/48fc32cc1aa01ba9ffa00add179ed22fed8f073d